experimental/oak_baremetal_app_qemu/src/hvm_start_info.rs

```diff
@@ -17,46 +17,65 @@
 use core::ffi::{c_char, CStr};
 use oak_baremetal_kernel::boot::{BootInfo, E820Entry, E820EntryType};
 
-include!(concat!(env!("OUT_DIR"), "/start_info.rs"));
+/// Magic value that needs to be present in the <StartInfo::magic> field.
+/// ("xEn3" with the 0x80 bit of the "E" set).
+#[cfg(not(feature = "multiboot"))]
+pub const BOOT_MAGIC: u32 = 0x336ec578;
 
-impl E820Entry for hvm_memmap_table_entry {
+#[repr(C)]
```

Comment:
Could you mention here or somewhere else the command line that you used to generate the baseline of this file via bindgen? It will be useful to have that around if / when we need to update things again.

Reply:
There is no command line per se (as we were generating the files in `build.rs`), but I've added comments stating where the data structures are from.